create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusHandler.java

```diff
@@ -1,24 +1,32 @@
 package no.sikt.nva.scopus;
 
-import static nva.commons.core.attempt.Try.attempt;
 import com.amazonaws.services.lambda.runtime.Context;
 import com.amazonaws.services.lambda.runtime.RequestHandler;
 import com.amazonaws.services.lambda.runtime.events.S3Event;
 import jakarta.xml.bind.JAXB;
-import java.io.StringReader;
-import java.net.URI;
 import no.scopus.generated.DocTp;
+import no.unit.nva.metadata.CreatePublicationRequest;
+import no.unit.nva.model.AdditionalIdentifier;
 import no.unit.nva.s3.S3Driver;
 import nva.commons.core.JacocoGenerated;
 import nva.commons.core.paths.UriWrapper;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import software.amazon.awssdk.services.s3.S3Client;
 
-public class ScopusHandler implements RequestHandler<S3Event, String> {
+import java.io.StringReader;
+import java.net.URI;
+import java.util.HashSet;
+import java.util.Set;
+
+import static nva.commons.core.attempt.Try.attempt;
+
+public class ScopusHandler implements RequestHandler<S3Event, CreatePublicationRequest> {
 
     public static final int SINGLE_EXPECTED_RECORD = 0;
     public static final String S3_URI_TEMPLATE = "s3://%s/%s";
+    public static final String ADDITIONAL_IDENTIFIERS_SCOPUS_ID_SOURCE_NAME = "ScopusId";
```

Comment:
Jeg er usikker hvor denne konstanten egentlig burde plasseres. Forslag?

Reply:
You could create classes that contain only final static fields and you could group those fields in categories. E.g. ScopusFields, CurrentlyUnmappedFields, etc.

Reply:
Jeg kunne tenke meg at det blir flere og flere slike Contanster. Kanskje et eget ScopusConstants klasse.

Reply:
Fikset